Amethyst/Managers/WindowManager.swift

```diff
@@ -523,44 +486,36 @@ final class WindowManager: NSObject, MouseStateKeeperDelegate {
         }
     }
 
-    func reevaluateWindows() {
```

Comment:
Has this been dead code for a while or was it re-implemented in a place I didn't see?

Reply:
Nevermind, found it